### PR TITLE
cool tech job first draft

### PR DESCRIPTION
**Pull request by ilanman**
Trying out this idea I had after presenting what my data team does. I found it helpful to couch it in terms of them performing the functions of a business analyst, PM, Eng and data analyst. Someone mentioned (paraphrase) "hey that sounds like a cool job". That sparked a post.

I'd appreciate guidance on structuring it in a coherent way, especially the intro and conclusion. And if you have other thoughts, feel free to add them!

### CODE REVIEW

File: content/post/data-analyst.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+<li>Exploring data and making some charts
+<li>Thinking about problems and experimenting with different models
+<li>Going down rabbit holes, and realizing 2 days were wasted going down that rabbit hole
+<li>Confidently providing recommendations and insights, usually with a polished presentation
+<li>Testing and validating those recommendations
+
+All of the above, and more, requires time and space to perform it, not dissimilar to what a researcher must do. Unlike researchers however, analysts need to ensure they are getting deliverables out the door. It's a fine balance to strike, which makes the job of an analyst all the more interesting.
+
+To perform analysis well and efficiently presumes, among other things, that tools are built (engineering), problems are scoped well enough (project managing), and the analyst has enough domain knowledge (business analysis) to work on the right kinds of problems. Else it really is just a research problem, which is why the analyst needs to have a strong working knowledge of all the above roles.
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/data-analyst.md]: You mean you don't also write in code? ;)

File: content/post/data-analyst.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+### Product/Project Manager
+
+Until the Data team gets larger, analysts must manage their workloads, scope projects, break up Epics into smaller tasks, prioritize and communicate with stakeholders. Analysts likely work in a matrix or pseudo-matrix organization, reporting to a Data lead but with dotted lines to the business. Even on a centralized Data team, there are likely several non-Data teams that use the result of the work produced by the analyst to make decisions. This means analysts need to balance competing priorities, communicate cross functionally, set expectations, and meet with stakeholders on a constant basis. Engineers have Product Managers who can help with all of these tasks, allowing them to focus purely on the code and architecture. 
+
+Some of this kind of work may be done by Analytics Managers, but a lot will still fall on the analyst's shoulders. PMing their own work does force analysts' to build up that muscle which will make their overall work product better in the end. They'll be working within the context of stakeholders, biasing their work to be of higher importance, than if they simply did their own work without regard to external teams.
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/data-analyst.md]: Agreed. The gist here is that working closer with stakeholders almost organically means that your work will be higher importance, all else equal. You're kind of starting ahead (on the "value meter") than someone who is working on something they think is valuable, before any input from stakeholders.

File: content/post/data-analyst.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+<li>Identify which business problems matter and which don't.
+<li>Understand how to quickly home in on the highest pain points, and know what's possible to fix and what isn't. This skill underestimated -- these folks are less likely to propose solutions that are out of sync with business reality.
+<li>Gain trust of their stakeholders, helping improve communication cross functionally and reduce coordination costs.
+
+Business analysts are helpful in ensuring that operations teams aren't always heads down, focused on executing; they can help folks step back and improve processes and exploit areas of opportunity. Management tends to appreciate this skill. 
+
+Analysts should be wary of spending too much time operating as business analysts -- signs to watch out for are: constantly putting out fires for the same stakeholder, spending more time in Excel and less in SQL and in general working on short term projects with urgent deadlines.
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/data-analyst.md]: I added some more content. Let me know if this addresses the downsides.

**Comment by ilanman** [comment]
@mikekaminsky @caitlinmoorman @sbreitenother Let me know if I've addressed your comments. 

@caitlinmoorman would you like to publish your 3 ladder posts each week? If so, I can delay mine until the week after your last one.

File: content/post/data-analyst.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Coolest Job in Tech"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
```
Comment (mikekaminsky) [inline content/post/data-analyst.md]: Don't forget to get Margarita started on this ;)

File: content/post/data-analyst.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+tags: ["Analytics", "Org Structure"]
+date: 2020-01-05T21:16:09-05:00
+draft: true
+---
+
+Data analysts at early stage, tech-adjacent start ups have the coolest jobs. For junior and mid level folks especially, there’s so much to learn (hopefully an exciting proposition!)  as analysts  wear many different hats. In addition, there are numerous career paths to take once an analyst has cut their teeth for a few years. Due to its broad set of responsibilities, an analyst’s job can be esoteric and often means different things to different people.
+
+Folks outside of Data sometimes struggle with answering the question, “What does Data do here?” Folks can reason around “analysis”, “reporting” and “insights”, but otherwise their understanding is usually surface level. I’ve found it helpful to explain what the Analytics team does in terms of other job functions with which they might be more familiar.
```
Comment (mikekaminsky) [inline content/post/data-analyst.md]: Folks twice scans oddly for me -- would change one to "People" or at least change the sentence structure so it's not the first word both times

File: content/post/data-analyst.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Coolest Job in Tech"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["Analytics", "Org Structure"]
+date: 2020-01-05T21:16:09-05:00
+draft: true
+---
+
+Data analysts at early stage, tech-adjacent start ups have the coolest jobs. For junior and mid level folks especially, there’s so much to learn (hopefully an exciting proposition!)  as analysts  wear many different hats. In addition, there are numerous career paths to take once an analyst has cut their teeth for a few years. Due to its broad set of responsibilities, an analyst’s job can be esoteric and often means different things to different people.
```
Comment (mikekaminsky) [inline content/post/data-analyst.md]: > as analysts  wear many different hats

i think 
> because analysts  wear many different hats

will be clearer

Comment (mikekaminsky) [reply content/post/data-analyst.md]: I'd change "In addition" to "even better" to emphasize the positive flow, and I'd add a "However" before "Due to..."

File: content/post/data-analyst.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+<li>Understand what’s feasible to fix. This skill can be underestimated – analysts are less likely to propose solutions that are out of sync with business reality.
+<li>Gain trust of their stakeholders, helping improve communication cross-functionally and reduce coordination costs.
+
+Business analysts ensure that operations teams aren’t singularly focused on executing. They are instrumental in helping teams step back and improve processes in order to  exploit areas of opportunity. While data analyst responsibilities often overlap with those of a business analyst, a good data analyst also build models, performs experiments, works cross functionally and occasionally spikes on more technical projects – tasks which all require time. Business analysts, however, operate on their business team’s schedule, requiring quick turn around and answers to very specific questions. A business analyst might answer the question,  “Why did sales drop last week?”, whereas a data analyst works  on a question like, “What is the single biggest driver within our control to increase sales?” The latter type of question could require weeks of work; the former perhaps a few hours.
```
Comment (mikekaminsky) [inline content/post/data-analyst.md]: >builds models

**Comment by caitlinmoorman** [comment]
@ilanman I currently have my next 2 articles scheduled for the next 2 Mondays, but I'm not beholden to that schedule if you want to rearrange.

**Comment by ilanman** [comment]
> @ilanman I currently have my next 2 articles scheduled for the next 2 Mondays, but I'm not beholden to that schedule if you want to rearrange.

Nope, that order works for me!

File: content/post/data-analyst.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+<li>Confidently providing recommendations and insights, usually with a polished presentation
+<li>Testing and validating those recommendations
+
+These exploratory tasks require time and space to think critically and creatively, not dissimilar to the work of a researcher. Unlike researchers however, analysts need to ensure they are getting deliverables out the door. It’s a fine balance to strike, which makes the job of an analyst all the more interesting.
+
+To perform analysis well and efficiently presumes that tools are built (engineering), problems are scoped (project management), and domain knowledge has been acquired (business analysis). A strong working knowledge of all the above job functions allows a Data Analyst to become more than a researcher, providing well-designed and tested insights and recommendations to the rest of the organization.
+
+## Where to next?
+
+Part of what makes being a data analyst a cool job are the potential future career paths. An experienced Data Analyst can move on to:
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/data-analyst.md]: Good call. Will try to make them pop more.

File: content/post/data-analyst.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+## Product/Project Manager
+
+Data analysts likely work in a [matrix or pseudo-matrix organization](https://towardsdatascience.com/what-is-the-most-effective-way-to-structure-a-data-science-team-498041b88dae), reporting to a Data lead but with dotted lines to the business. Even on a centralized Data team, there are likely several non-Data teams that use the result of the work produced by the analyst to make decisions. This means that data analysts need to manage their own workloads, scope their projects, balance competing priorities, communicate cross-functionally to set expectations, and meet with stakeholders consistently. Engineers usually have Product Managers who can help with all of these tasks, allowing them to focus purely on the code and architecture.
+
+Project management skills improve the quality of work produced by data analysts, as does constant communication directly with stakeholders. But these responsibilities can also add a lot of administrative overhead -- more meetings, documentation and emails, all of which take away from the core job of an analyst – analyzing!
+
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/data-analyst.md]: great point. I touch upon that in the business analyst section, but I called it out here too.

File: content/post/data-analyst.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+Data analysts who spend time writing code and perfecting software engineering practices must balance these responsibilities against stepping back and thinking holistically about the business problem. Analysts are better equipped to write quality code only if they are spending enough time understanding their core stakeholders’ problems.
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/data-analyst.md]: that's a good point too! Will add in something around over-engineering.

**Comment by ilanman** [comment]
@caitlinmoorman @mikekaminsky 

Had some merge conflicts so you can ignore all the commits except the second to last (153aa8da1af67a3c85a6d95f8cf7fc17799a3cb0).

I'm taking a different approach. Instead of "data analysis is cool", its more "data analysis is a great job to give yourself opportunities for career growth, and here are a few ways analysts should be thinking about their roles". I think it's more useful for people (thanks for the tip Caitlin).

And thanks again (and in advance) for the edits, hopefully it's close!

File: content/post/career-ladders-part-1.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
+---
```
Comment (mikekaminsky) [inline content/post/career-ladders-part-1.md]: @ilanman should you rebase master to not accidentally deploy these?

File: content/post/data-analyst.md [hunk 12]
```diff
@@ -0,0 +1,74 @@
+---
+title: "The Analyst career path"
```
Comment (mikekaminsky) [inline content/post/data-analyst.md]: Title-casing is odd here

Comment (mikekaminsky) [reply content/post/data-analyst.md]: And also maybe not a great title for this post -- something like "The many hats of a data analyst" conveys better what's actually discussed here

File: content/post/data-analyst.md [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["Analytics", "Org Structure"]
+date: 2020-01-26T21:16:09-05:00
+draft: true
+---
+
+Data analysts at early stage, tech-adjacent start ups have unique jobs that allow you to work on a bunch of different things which then opens up a ton of opportunities for career progression. However because of an analyst's broad set of responsibilities, folks often struggle with understanding what exactly an analyst does. I’ve found it helpful to explain what the Analytics team does in terms of other job functions with which people might be more familiar.
```
Comment (mikekaminsky) [inline content/post/data-analyst.md]: I think you could re-write this to shift the emphasis to better match the bulk of the post. Something like

> I've often found that it's difficult to explain to business stakeholders what exactly a data analyst on my analytics team does. And even beyond that, I've sometimes found it difficult to explain to the data analysts _themselves_ why their job is so valuable to the business and what their future career opportunities might look like.

> Data analysts at early stage, tech-adjacent start ups have unique jobs that allow you to work on a bunch of different things which then opens up a ton of opportunities for career progression, and in this post I'll talk about all of the different roles data analysts play and what that means for future opportunities.

File: content/post/career-ladders-part-1.md (hunk 11 above)
Comment (ilanman) [reply content/post/career-ladders-part-1.md]: Github is giving me some trouble. Will probably squash my commits and just commit the latest change.

**Pull request merged**